openid-4-verifiable-presentations-1_0.md

```diff
@@ -710,6 +720,93 @@ The rules for selecting claims to send are defined in (#selecting_claims).
 
 Note that multiple Credential Queries in a request MAY request a presentation of the same Credential.
 
+### Trusted Authorities Query {#dcql_trusted_authorities}
+
+A Trusted Authorities Query is an object representing information that helps to identify an authority
+or the trust framework that certifies Issuers. A Credential is identified as a match
+to a Trusted Authorities Query if it matches with one of the provided values in one of the provided
+types. How exactly the matching works is defined for the different types below
+
+Note that direct Issuer matching can also work using claim value matching if supported (e.g., value matching
+the `iss` claim in an SD-JWT) if the mechanisms for `trusted_authorities` are not applicable but might
+be less likely to work due to the constraints on value matching (see #selecting_claims for more details).
+
+Each entry in `trusted_authorities` MUST be an object with the following properties:
+
+`type`:
+: REQUIRED. A string uniquely identifying the type of information about the issuer trust framework.
+Types defined by this specification are listed below.
+
+`values`:
+: REQUIRED. An array of strings, where each string (value) contains information specific to the
+used Trusted Authorities Query type that allows to identify an issuer, trust framework, or a federation that an
+issuer belongs to.
+
+Below are descriptions for the different Type Identifiers (string), the description on how to interpret
+and perform the matching logic for each provided value.
+
+Note that depending on the trusted authorities type used, the underlying mechanisms can have
+different privacy implications. More detail on privacy considerations for the trusted authorities
+can be found in (#privacy_trusted_authorities).
+
+#### Authority Key Identifier
+
+Type:
+: `"aki"`
+
+Value:
+: Contains the KeyIdentifier of the AuthorityKeyIdentifier as defined in Section 4.2.1.1 of [@!RFC5280],
+encoded as base64url. The raw byte representation of this element MUST match with the AuthorityKeyIdentifier
+element of an X.509 certificate in the certificate chain present in the credential (e.g., in the header of
+an mdoc or SD-JWT). Note that the chain can consist of a single certificate.
```

Comment:
```suggestion
: Contains the KeyIdentifier of the AuthorityKeyIdentifier as defined in Section 4.2.1.1 of [@!RFC5280],
encoded as base64url. The raw byte representation of this element MUST match with the AuthorityKeyIdentifier
element of an X.509 certificate in the certificate chain of the credential signing certificate. Note that the chain can consist of a single certificate, and the credential can include the entire X.509 chain or parts of it.
```

Reply:
I mean the whole chain. For mdocs you have x5chain = doc signer, and the IACA is transmitted out of band. The AKI could also match the IACA, not only the IACA if this makes sense.